client/web/compose/src/components/PageBlocks/Wrap/Card.vue

```diff
@@ -25,11 +25,11 @@
             </h5>
 
             <div
-              v-if="block.options.refreshRate >= 5 || hasMagnifyIcon"
+              v-if="magnifyOrRefresh"
```

Comment:
We should come up with a different name for this, since if we add another option then you'll have to change this name aswell. Something like showExtraOptions